### ISSUE TITLE
Ability to multi select columns and reorder on the fly.

### ISSUE THREAD

**Issue by shaun-ba** [sqlitebrowser/sqlitebrowser#2304]
Describe the new feature
--------------------------

So based on the fact this is an editing tool, it would be really useful to be able to move around the columns when editing, rather than having to modify the whole table and move things up and down.

I have say 25 columns, and would like to move them around easily and hide ones that aren't often used (by selecting multiple at a time and choosing to hide). As if I am to edit a bunch of cells the columns need to be next to each other, or there is a lot of horizontal scrolling!

Current version: Version 3.12.99 (Jun 18 2020)

**Comment by chrisjlocke** [comment]
Takes a bit more work (triggers, et al) but you could create a view with the fields you wanted in the order you wanted.  Db4S does support editing in views.  Just not 'immediately' ... ie, you have to jump some hurdles, but that's only because views can link through several tables.

**Comment by shaun-ba** [comment]
With so many tables views is just a bit of a faff, I just thought for a UI that's an editor it would be useful to implement.

Functionality just to be similar to a spreadsheet with column selection etc

**Comment by chrisjlocke** [comment]
It's a good idea, but I don't know how easily it could be implemented.  We'll add it to the list though. 😃

**Comment by justinclift** [comment]
Agreed, this is a decent idea.  Probably wouldn't be too hard to implement the column reordering, and the saving/loading for persistence to project files.
